.golangci.yml

```diff
@@ -82,32 +82,26 @@ linters-settings:
   wrapcheck:
     ignoreSigs:
       - github.com/omni-network/omni/
+      - google.golang.org/grpc/status # No point wrapping gRPC/network errors.
 
 issues:
   fix: true
   exclude-rules:
-    - path: 'metrics.go'
-      linters:
-        - gochecknoglobals # Promauto metrics are global variables
```

Comment:
removed `gochecknoglobals` in general since we have a ton of globals in many places